tsdb/store.go

```diff
@@ -458,16 +443,24 @@ func (s *Store) DeleteRetentionPolicy(database, name string) error {
 	}
 
 	// Remove the retention policy folder from the the WAL.
-	return os.RemoveAll(filepath.Join(s.EngineOptions.Config.WALDir, database, name))
+	if err := os.RemoveAll(filepath.Join(s.EngineOptions.Config.WALDir, database, name)); err != nil {
```

Comment:
does the RP get removed from the metadata before this is called? Would this maybe cause a problem if it isn't?